userguide/system.rst

```diff
@@ -2375,8 +2375,38 @@ To generate a support ticket, fill in the fields:
   diagnosing a performance bottleneck.
 
 * **Environment** is a drop-down menu to indicate the role of the
-  affected system. Choices are *Production*, *Staging*, *Test*,
-  *Prototyping*, or *Initial Deployment/Setup*.
+  affected system.
+
+  .. tabularcolumns:: |>{\RaggedRight}p{\dimexpr 0.20\linewidth-2\tabcolsep}
+                      |>{\RaggedRight}p{\dimexpr 0.16\linewidth-2\tabcolsep}
+
+.. _environment options:
+
+.. table:: Environment Options
+      :class: longtable
+
+   +---------------------+----------------------------------------------------------+
+   | Environment         | Description                                              |
+   +=====================+==========================================================+
+   | Production          | The system is being used for production. Data loss       |
+   |                     | affects customers directly.                              |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Staging             | The system is being prepared for production.             |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Test                | The system is being used internally.                     |
+   |                     | The system does not affect customers.                    |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Prototyping         | The system is unique. The use case is likely             |
+   |                     | to be rare or never seen before.                         |
+   |                     |                                                          |
```

Comment:
The definition for "Prototyping" in this context makes it seem like the system is "unique" or is an "alpha" model of the machine. This needs to be discussed further as I don't believe this section makes much sense.

This section is probably referring to "Proof of Concept". However, this section should be removed until the original intent can be understood.

Reply:
Those are my thoughts as well. I also thought that the "Prototyping" environment would be very similar to the "Testing" environment. Thus, if support doesn't have a concrete explanation of what the environment means and it's similar to "Testing", I think we can remove the option from the UI entirely.

Reply:
Agreed.  The overlap between Testing and Prototyping is only going to confuse the user unless we add concrete descriptions of what each means.